### PR TITLE
nfs-ganesha: change CMAKE_BUILD_TYPE to Maintainer

### PR DESCRIPTION
**Pull request by alimaredia**
Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

**Comment by alimaredia** [comment]
@ajarr on #ganesha IRC channel,  Frank told me that "Maintainer" should be used for CI builds. This PR will change both the shaman and download.ceph.com rpms to be built that way.

File: nfs-ganesha-stable/build/build_rpm [hunk 1]
```diff
@@ -12,6 +12,9 @@ RELEASE="$(lsb_release --short -r | cut -d '.' -f 1)" # sytem release
 # Make sure old rpms are not leftover on the system
 sudo yum remove -y librados2 librgw2 libcephfs2 librados-devel librgw-devel libcephfs-devel
 
+# Make sure old repo metadata are not leftover on the system
+sudo yum clean all
```
Comment (alimaredia) [inline nfs-ganesha-stable/build/build_rpm]: @ajarr this should fix the issue with the ceph related code not being built.

File: nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml [hunk 2]
```diff
@@ -76,7 +76,7 @@
       - string:
           name: CEPH_VERSION
           description: "The version of Ceph to specify for installing ceph libraries"
-          default: "12.2.2"
+          default: "12.2.3"
```
Comment (alimaredia) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: @ajarr will the latest stable version of luminous still be compatible with 2.5.5?

Comment (ajarr) [reply nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: Yes. It should.

**Pull request merged**